docs/qase.md

```diff
@@ -0,0 +1,21 @@
+## [Qase-Reporter](../pkg/qase/README.md)
+## [Qase-Patch-Validation-Create](#qase-patch-validation-create)
+
+
+# Qase-Patch-Validation-Create
+
+### Description
+This job aims to automatically create a patch validation in Qase for a given project and test plan ID.
```

Comment:
`This job aims to automatically create Test Runs in Qase to perform distros patch/minor validation for a given project ID and test plan ID.`